src/Workspaces/Core/Portable/Workspace/Solution/SolutionCompilationState.RegularCompilationTracker_Generators.cs

```diff
@@ -396,7 +396,7 @@ static GeneratorDriver CreateGeneratorDriver(ProjectState projectState)
                 return compilationFactory.CreateGeneratorDriver(
                     projectState.ParseOptions!,
                     GetSourceGenerators(projectState),
-                    projectState.AnalyzerOptions.AnalyzerConfigOptionsProvider,
+                    projectState.ProjectAnalyzerOptions.AnalyzerConfigOptionsProvider,
```

Comment:
Source generators shouldn't use fallback?

Reply:
No, there is no such thing as a host-installed source generator. All source generators in all cases must ignore host options.